src/utils/httpVariableProviders/environmentVariableProvider.ts

```diff
@@ -49,6 +50,27 @@ export class EnvironmentVariableProvider implements HttpVariableProvider {
         const variables = this._settings.environmentVariables;
         const currentEnvironmentVariables = variables[environmentName];
         const sharedEnvironmentVariables = variables[EnvironmentController.sharedEnvironmentName];
+        this.mapEnvironmentVariables(currentEnvironmentVariables, sharedEnvironmentVariables);
         return Object.assign({}, sharedEnvironmentVariables, currentEnvironmentVariables);
     }
+
+    private mapEnvironmentVariables(current: any, shared: any) {
+        for (let key in current) {
+            const value = current[key];
+            if (typeof(value) !== "string") {
+                continue;
+            }
+            const variableRegex = /\{{2}\$shared (.+?)\}{2}/;
+            const match = variableRegex.exec(value);
+            if (!match) {
+                continue;
+            }
+            const referenceKey = match[1].trim();
+            const referenceValue = shared[referenceKey];
```

Comment:
How about directly using `in`  operator to test if the given key in object like following:
```js
const referenceKey = match[1].trim();
if (referenceKey in shared) {
    current[key] = shared[referenceKey];
}
```

Reply:
Alright I've made these changes. However I've just noticed this doesn't update the breadcrumbs yet:
![image](https://user-images.githubusercontent.com/53242951/63743625-2aee4a80-c8e0-11e9-89a5-af2feec9ee7c.png)
I'll work on this now.